build.gradle.kts

```diff
@@ -0,0 +1,161 @@
+import org.jetbrains.kotlin.gradle.tasks.KotlinCompile
+
+plugins {
+    kotlin("jvm") version "1.6.20"
+    id("com.github.johnrengelman.shadow") version "7.1.2"
+}
+
+group = "com.discordsrv"
+version = "1.26.1-SNAPSHOT"
+val minecraftVersion = "1.16.5"
+val targetJavaVersion = 1.8
+
+java {
+    val javaVersion = JavaVersion.toVersion(targetJavaVersion)
+    sourceCompatibility = javaVersion
+    targetCompatibility = javaVersion
+}
+
+tasks {
+    processResources {
```

Comment:
These tasks don't normally need to be wrapped in `tasks {}`, unless that's something specific to kotlin script for some reason

Reply:
> that's something specific to kotlin script for some reason

It is, as far as I can tell, could be wrong